server/routes/detailsOfOffence/detailsOfOffence.ts

```diff
@@ -21,11 +21,11 @@ export default class DetailsOfOffenceRoutes {
     const { draftAdjudication, incidentRole, prisoner, associatedPrisoner } =
       await this.decisionTreeService.draftAdjudicationIncidentData(adjudicationNumber, user)
     const allOffences = await this.helper.populateSessionIfEmpty(adjudicationNumber, req, res)
+    const isYouthOffender = draftAdjudication.isYouthOffender || false
```

Comment:
This class and the related njk really need an overhaul as it does lots of things it doesn't need to when there are no offences.

I'll make another ticket to sort this out because it can be made far simpler

Reply:
sure.  i just thought i would keep it consistent, it was using the null value lower down.   why i moved this line so it was in scope